packages/storybook-react/src/stories/RadioButton/RadioButtonStates.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { RadioButton } from '@utrecht/component-library-react';
+import React from 'react';
+import { argTypes } from './argTypes';
+
+export default {
+  title: 'React.js Component/Radio Button/States',
+  id: 'react-radio-button-states',
+  component: RadioButton,
+  argTypes,
+} as ComponentMeta<typeof RadioButton>;
+
+const Template: ComponentStory<typeof RadioButton> = (args) => <RadioButton {...args} />;
+
+export const Hover = Template.bind({});
+
+Hover.parameters = {
+  pseudo: { hover: true },
+};
+
+export const Focus = Template.bind({});
+
+Focus.parameters = {
+  pseudo: { focus: true },
+};
+
+export const FocusVisible = Template.bind({});
+
+FocusVisible.parameters = {
+  pseudo: { focusVisible: true },
```

Comment:
I think for a realistic version we should always use `focus` together with `focusVisible`:

```js
{
  pseudo: {
    focus: false,
    focusVisible: true,
  },
}
```

We should probably also look back at previous code to fix this for other pseudo states.